app/(routes)/pastEvents/[eventId]/photoAlbum/page.tsx

```diff
@@ -1,9 +1,30 @@
-import { NextPageContext } from 'next';
-
 import Wrapper from '@/components/layout/Wrapper';
 import { getEventPhotoAlbum } from '@/app/service/meetup/MeetupService';
+import { Inter } from 'next/font/google'
+import Gallery from '@/components/Gallery/Gallery';
+
+const inter = Inter({ 
+  subsets: ['latin'],
+  weight: ['700'],
+})
+
+const parseDate = (dateString:String | undefined) => {
```

Comment:
Two things here.

1. TS syntax nit-pick: can represent the undefined with a question mark, like `(dateString?: string)`

2. Move this function to the Date Utils file in the utils folder so that it can reuse later if needed... There may already be some code in there to help with this